### ISSUE TITLE
Superfluous inactive back button

### ISSUE THREAD

**Issue by bertob** [GustavoPeredo/Font-Downloader#69]
So far as I can tell this button is always inactive. What is it for? Can we just remove it?

**Comment by GustavoPeredo** [comment]
This button is useful when it is in "smartphone" mode. I chose not to make the transition a but more seamless between "desktop" and "mobile" mode:

![Bildschirmvideo von 12 10 2021 22_37_35](https://user-images.githubusercontent.com/30641461/137025978-68c2a575-7bd9-4945-9303-8a38cf2c56eb.gif)

**Comment by bertob** [comment]
Aha, in that case the solution is to just hide it instead of making it inactive when the leaflet is unfolded. That's what we do in all other apps.

**Comment by GustavoPeredo** [comment]
That's true, it's surprising how I never noticed that. I'll take some time to update this then!
